lib/streamlit/config.py

```diff
@@ -252,6 +252,15 @@ def _global_unit_test():
     default_val=10 * 1e3)  # 10k
 
 
+_create_option(
+    'global.maxCachedMessageAge',
+    description='''Expire cached ForwardMsgs whose age is > this value.
+        A message's age is defined by how many times its report has finished
+        running since the message has been accessed.''',
```

Comment:
Replace "report" with "script". We're retiring the name "report" and will use "script" for the code and "app" for the thing in the browser.